src/Test/WebTestCase.php

```diff
@@ -26,17 +25,12 @@ abstract class WebTestCase extends KernelTestCase
     /**
      * {@inheritdoc}
      */
-    protected static function createClient(array $server = []) : Client
+    protected static function createClient(array $server = []) : HttpKernelBrowser
```

Comment:
If we do not typehint against `Symfony\Bundle\FrameworkBundle\KernelBrowser` we would not really depend on `symfony/framework-bundle` for core functionality (of course it would be needed to run the console commands, but it's not required.

But not sure if I am missing something :)

Reply:
I just checked the usage of FrameworkBundle and `Speicher210\FunctionalTestBundle\Test\KernelTestCase` extends `Symfony\Bundle\FrameworkBundle\Test\KernelTestCase` so we should actually move `symfony/framework-bundle` to "required" anyway :)

Reply:
I changed all the type hints and added `symfony/framework-bundle` to composer required